static/app/components/tabs/index.stories.tsx

```diff
@@ -225,6 +225,50 @@ export default storyBook(Tabs, story => {
           </TabPanels>
         </Tabs>
       </SizingWindow>
+      <br />
+      <Matrix<TabsProps<string> & TabListProps>
+        render={props => (
+          <Tabs orientation={props.orientation}>
+            <TabList variant={props.variant}>
+              {TABS.map(tab => (
+                <TabList.Item key={tab.key}>{tab.label}</TabList.Item>
+              ))}
+            </TabList>
+            <TabPanels>
+              {TABS.map(tab => (
+                <TabPanels.Item key={tab.key}>{tab.content}</TabPanels.Item>
+              ))}
+            </TabPanels>
+          </Tabs>
+        )}
+        propMatrix={{
+          orientation: ['horizontal', 'vertical'],
+          variant: ['flat', 'filled', 'floating'],
```

Comment:
Can also include `undefined` if that's the default value, so people know they can skip the prop if they want.

```suggestion
          orientation: [undefined, 'horizontal', 'vertical'],
          variant: [undefined, 'flat', 'filled', 'floating'],
```

Reply:
And then actually, if you wanted to it's possible to merge the two `propMatrix` values so it's not typed out twice:

```
const propMatrix = {
  hideBorder: [undefined, false, true],
  orientation: [undefined, 'horizontal', 'vertical'],
  variant: [undefined, 'flat', 'filled', 'floating'],
};

return <Matrix
  propMatrix={propMatrix}
  selected={['hideBorder', 'variant']}
  ...
/>
```